flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+  description =
+    "Tool to simplify secure shell connections over AWS Systems Manager (SSM).";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs?ref=nixpkgs-unstable";
```

Comment:
I'm a bit leery of any configuration using an unstable branch. Is this necessary for some reason?

Reply:
not too big of a deal but I've updated the channel. Please see fcbd888841c3a2b2ed3dc639f40ff0260db3d44e